2024-10-25 15:04:05,770 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:05:31,046 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:13:47,816 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:21:08,977 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:37:11,411 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:41:24,832 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-25 15:42:51,760 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 13:07:05,422 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 13:41:15,836 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 13:42:49,818 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 14:13:26,688 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 14:13:36,686 ERROR: Exception on /index [GET] [in /home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py:838]
Traceback (most recent call last):
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/home/obeezy/Desktop/PythonProjects/microblog/app/routes.py", line 31, in index
    posts = db.session.scalars(current_user.following_posts(), page=page,
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/sqlalchemy/orm/scoping.py", line 1908, in scalars
    return self._proxied.scalars(
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2470, in scalars
    return self._execute_internal(
TypeError: _execute_internal() got an unexpected keyword argument 'page'
2024-10-29 14:21:06,059 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-29 14:50:03,257 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:44]
2024-10-30 15:33:03,732 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 15:34:43,591 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 16:34:38,974 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 16:34:50,087 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 16:35:20,402 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 16:35:52,606 ERROR: Exception on /reset_password_request [GET] [in /home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py:838]
Traceback (most recent call last):
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/obeezy/Desktop/PythonProjects/microblog/app/routes.py", line 176, in reset_password_request
    return render_template('reset_password_request.html',
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1084, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1013, in get_template
    return self._load_template(name, globals)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 972, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: reset_password_request.html
2024-10-30 16:40:55,342 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
2024-10-30 16:41:14,627 ERROR: Exception on /reset_password_request [POST] [in /home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py:838]
Traceback (most recent call last):
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/obeezy/Desktop/PythonProjects/microblog/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/obeezy/Desktop/PythonProjects/microblog/app/email.py", line 18, in send_password_reset_email
    text_body=render_template('email/reset_password.txt',
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1084, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1013, in get_template
    return self._load_template(name, globals)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 972, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/home/obeezy/Desktop/PythonProjects/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: email/reset_password.txt
2024-10-30 16:57:47,111 INFO: Microblog startup [in /home/obeezy/Desktop/PythonProjects/microblog/app/__init__.py:46]
